### PR TITLE
[TRANSLATION] Update Hebrew (he-IL) translation for several applications

### PR DESCRIPTION
**Pull request by amir16yp**
## Purpose

- Added various Hebrew translations for FontView, Notepad and RAPPS that were mssisng

## TODO

- Nothing more yet

### CODE REVIEW

**Comment by gonzoMD** [comment]
cc @peterooch for review, please.

**Comment by binarymaster** [comment]
Cc @reactos/lang-hebrew

**Review by binarymaster — CHANGES_REQUESTED** [review]
Hello,
Thanks for your interest in ReactOS!
We ask our contributors [to use their full name when committing](https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations).
[Please amend your commit with your full name](https://stackoverflow.com/a/3042512), and update this PR.

To change this for future PR's, you can update it for globally (for the system):
```
git config --global user.name "Your Name"
git config --global user.email you@example.com
```

Or just for the current repository:
```
git config user.name "Your Name"
git config user.email you@example.com
```

Also please set up your GitHub account accordingly, so maintainers can use Squash & Merge button without getting the author field modified back to nickname by GitHub, it's done in two steps:

1. Set your full name in [GitHub profile settings](https://github.com/settings/profile) (it should match your `user.name` setting)
1. Uncheck `Keep my email addresses private` in [GitHub email settings](https://github.com/settings/emails)

Regards,

**Review by JoachimHenze — CHANGES_REQUESTED** [review]

File: base/applications/notepad/lang/he-IL.rc [hunk 1]
```diff
@@ -157,15 +157,18 @@ BEGIN
     STRING_TEXT_FILES_TXT "קבצי טקסט (*.txt)"
     STRING_TOOLARGE "File '%s' is too large for notepad.\n\
 Please use a different editor."
-    STRING_NOTEXT "You didn't enter any text. \
-\nPlease type something and try again"
-    STRING_DOESNOTEXIST "File '%s'\ndoes not exist\n\n\
-Do you want to create a new file ?"
-    STRING_NOTSAVED "File '%s'\nhas been modified\n\n\
-Would you like to save the changes ?"
+    STRING_TOOLARGE "הקובץ '%s' גדול מידי עבור פנקס הרשימות\n\
```
Comment (binarymaster) [inline base/applications/notepad/lang/he-IL.rc:160]: `STRING_TOOLARGE` is duplicated (you translated it, but didn't removed original english string)

File: base/applications/notepad/lang/he-IL.rc [hunk 2]
```diff
@@ -157,15 +157,18 @@ BEGIN
     STRING_TEXT_FILES_TXT "קבצי טקסט (*.txt)"
     STRING_TOOLARGE "File '%s' is too large for notepad.\n\
 Please use a different editor."
```
Comment (binarymaster) [inline base/applications/notepad/lang/he-IL.rc:159]:
```suggestion
```

File: base/applications/notepad/lang/he-IL.rc [hunk 3]
```diff
@@ -155,17 +155,18 @@ BEGIN
...
-Do you want to create a new file ?"
-    STRING_NOTSAVED "File '%s'\nhas been modified\n\n\
-Would you like to save the changes ?"
+    STRING_TOOLARGE "הקובץ '%s' גדול מידי עבור פנקס הרשימות\n\
+בבקשה השתמש בעורך טקסט אחר."
+    STRING_NOTEXT "לא הקלדת טקסט. \
+\nאנא הקלד טקסט ונסה שנית"
+    STRING DOESNOTEXIST "הקובץ '%s'\nלא קיים\n\n\
+האם ברצונך ליצור קובץ חדש?"
+
```
Comment (binarymaster) [inline base/applications/notepad/lang/he-IL.rc:164]:
```suggestion
```

File: base/applications/notepad/lang/he-IL.rc [hunk 4]
```diff
@@ -155,17 +155,17 @@ BEGIN
...
-Please use a different editor."
-    STRING_NOTEXT "You didn't enter any text. \
-\nPlease type something and try again"
-    STRING_DOESNOTEXIST "File '%s'\ndoes not exist\n\n\
-Do you want to create a new file ?"
-    STRING_NOTSAVED "File '%s'\nhas been modified\n\n\
-Would you like to save the changes ?"
+    STRING_TOOLARGE "הקובץ '%s' גדול מידי עבור פנקס הרשימות\n\
+בבקשה השתמש בעורך טקסט אחר."
+    STRING_NOTEXT "לא הקלדת טקסט. \
```
Comment (JoachimHenze) [inline base/applications/notepad/lang/he-IL.rc:160]:
```suggestion
    STRING_NOTEXT "לא הקלדת טקסט.\
```

Comment (binarymaster) [reply base/applications/notepad/lang/he-IL.rc:160]: ^ this one could be fixed later, all translations are affected: https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=STRING_NOTEXT

File: base/applications/rapps/lang/he-IL.rc [hunk 5]
```diff
@@ -204,20 +204,20 @@ BEGIN
     IDS_APPLICATIONS "יישומים"
     IDS_CHOOSE_FOLDER_TEXT "בחר תקייה שתשמור הורדות:"
     IDS_CHOOSE_FOLDER_ERROR "התקייה שהזנת לא קיימת, ליצור אותה?"
-    IDS_URL_INVALID "The URL you have specified is invalid or not supported. Please correct it!"
+    IDS_URL_INVALID "כתובת האתר שציינת אינה חוקית או שאינה נתמכת. בבקשה תתקן זאת!"
     IDS_APP_REG_REMOVE "Are you sure you want to delete the data on the installed program from the registry?"
```
Comment (JoachimHenze) [inline base/applications/rapps/lang/he-IL.rc:208]: Is there any specific reason why you decided to not translate all remaining english strings in this file?

File: base/applications/notepad/lang/he-IL.rc (hunk 4 above)
Comment (JoachimHenze) [reply base/applications/notepad/lang/he-IL.rc:160]: Yes, I do concur. Can be done in a separate PR also.

File: base/applications/notepad/lang/he-IL.rc [hunk 6]
```diff
@@ -155,17 +155,17 @@ BEGIN
...
+\nאנא הקלד טקסט ונסה שנית"
+    STRING DOESNOTEXIST "הקובץ '%s'\nלא קיים\n\n\
```
Comment (binarymaster) [inline base/applications/notepad/lang/he-IL.rc:162]:
```suggestion
    STRING_DOESNOTEXIST "הקובץ '%s'\nלא קיים\n\n\
```

File: base/applications/notepad/lang/he-IL.rc [hunk 7]
```diff
@@ -155,17 +155,17 @@ BEGIN
...
+האם ברצונך ליצור קובץ חדש?"
+    STRING NOTSAVED "הקובץ '%s'\nשונה\n\n\
```
Comment (binarymaster) [inline base/applications/notepad/lang/he-IL.rc:164]:
```suggestion
    STRING_NOTSAVED "הקובץ '%s'\nשונה\n\n\
```

**Review by binarymaster — COMMENTED** [review]
Fix some broken string identifiers.

File: base/applications/rapps/lang/he-IL.rc (hunk 5 above)
Comment (amir16yp) [reply base/applications/rapps/lang/he-IL.rc:208]: I didn't quite know how to translate the word "Registry" into Hebrew
Might have to look into how windows calls it

**Review by JoachimHenze — APPROVED** [review]

File: base/applications/rapps/lang/he-IL.rc (hunk 5 above)
Comment (HBelusca) [reply base/applications/rapps/lang/he-IL.rc:208]: Note that there are other untranslated strings in this file, as well as in the fontview one too.

File: base/applications/notepad/lang/he-IL.rc [hunk 8]
```diff
@@ -176,15 +176,15 @@ task.\nClose one or more applications to increase the amount of\nfree memory."
...
-    STRING_LINE_NUMBER_OUT_OF_RANGE "The specified line number is out of range."
-    STRING_NOWPRINTING "Now printing page..."
-    STRING_PRINTCANCELING "The print job is being canceled..."
-    STRING_PRINTCOMPLETE "Printing is successfully done."
-    STRING_PRINTCANCELED "Printing has been canceled."
-    STRING_PRINTFAILED "Printing is failed."
+    STRING_LINE_NUMBER_OUT_OF_RANGE "מספר העמודים הניתן מחוץ לטווח החוקי."
+    STRING_NOWPRINTING "מדפיס דף עכשיו..."
+    STRING_PRINTCANCELING "ביצוע ההדפסה מבוטל כעת..."
+    STRING_PRINTCOMPLETE "הדפסה בוצעה בהצלחה."
```
Comment (peterooch) [inline base/applications/notepad/lang/he-IL.rc:184]: הסתיימה

File: base/applications/rapps/lang/he-IL.rc [hunk 9]
```diff
@@ -204,20 +204,20 @@ BEGIN
...
     IDS_UNABLE_TO_REMOVE "Unable to remove data on the program from the registry!"
-    IDS_UNABLE_TO_INSTALL "Unable to open installer!"
+    IDS_UNABLE_TO_INSTALL "לא ניתן לפתוח את התקנת התוכנה!"
     IDS_UNABLE_TO_QUERY_CERT "Unable to retrieve certificate info.\n\nDo you want to continue anyway?"
-    IDS_INTEG_CHECK_TITLE "Verifying package integrity…"
-    IDS_INTEG_CHECK_FAIL "The package did not pass the integrity check, it may have been corrupted or tampered with during downloading. Running the software is not recommended."
-    IDS_INTERRUPTED_DOWNLOAD "The download was interrupted. Check connection to Internet."
-    IDS_UNABLE_TO_WRITE "Unable to write to disk. Disk may be at capacity."
+    IDS_INTEG_CHECK_TITLE "מוודא את שלמות התוכנה..."
+    IDS_INTEG_CHECK_FAIL "התוכנה לא עברה את וידוי השלמות, יתכן כי נפגמה או הוחלפה במהלך ההורדה. הרצת התוכנה אינה מומלצת."
```
Comment (peterooch) [inline base/applications/rapps/lang/he-IL.rc:216]: אולי חבילה יהיה יותר טוב? לא כזה משנה

File: base/applications/rapps/lang/he-IL.rc [hunk 10]
```diff
@@ -204,20 +204,20 @@ BEGIN
...
     IDS_CHOOSE_FOLDER_TEXT "בחר תקייה שתשמור הורדות:"
     IDS_CHOOSE_FOLDER_ERROR "התקייה שהזנת לא קיימת, ליצור אותה?"
-    IDS_URL_INVALID "The URL you have specified is invalid or not supported. Please correct it!"
+    IDS_URL_INVALID "כתובת האתר שציינת אינה חוקית או שאינה נתמכת. בבקשה תתקן זאת!"
     IDS_APP_REG_REMOVE "Are you sure you want to delete the data on the installed program from the registry?"
     IDS_INFORMATION "מידע"
-    IDS_UNABLE_TO_DOWNLOAD "Unable to download the package! Address not found!"
-    IDS_UNABLE_TO_DOWNLOAD2 "Unable to download the package! Check Internet Connection!"
+    IDS_UNABLE_TO_DOWNLOAD "לא ניתן להוריד את התוכנה! בדוק את תקינות הכתובת"
+    IDS_UNABLE_TO_DOWNLOAD2 "לא ניתן להוריד את תוכנה! בדוק את חיבורך לרשת"
```
Comment (peterooch) [inline base/applications/rapps/lang/he-IL.rc:211]: אינטרנט, "רשת" זה כללי מידי

**Comment by binarymaster** [comment]
Please resolve conflicts (master branch has been updated).

image

**Comment by amir16yp** [comment]
uhh idk what i just did

**Comment by binarymaster** [comment]
You've just reset your branch to upstream/master without keeping PR commits.

**Pull request closed without merging**